kiwi/xml_parse.py

```diff
@@ -3,7 +3,7 @@
 
 #
 # Generated  by generateDS.py version 2.29.14.
-# Python 3.6.5 (default, Mar 31 2018, 19:45:04) [GCC]
+# Python 3.7.2 (default, Dec 30 2018, 16:18:15) [GCC]
```

Comment:
Were those changes done manually? I'd have expected them to be autogenerated by calling `make kiwi/schema/kiwi.rng` in which case I'd also expect it to modify the project path... Anyway, those are not relevant changes, so I would just omit any change here and keep the original file.

Reply:
I'll drop that, I just thought to bump the Python version string in that file so that we appear to be using the newest stuff :wink: